src/vivarium_cluster_tools/psimulate/utilities.py

```diff
@@ -139,6 +139,17 @@ def get_valid_project(project, cluster):
     return project
 
 
+def get_valid_queue(max_runtime):
+    hours, minutes, seconds = max_runtime.split(":")
```

Comment:
Does something ensure that you have all 3 elements (hours, minutes, seconds)?

Reply:
we don't. click has support for validation callbacks, this is a good place to start